catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -667,28 +666,12 @@
 	private List<Brick> setupAssertionsCategoryList(Context context) {
 		List<Brick> assertionsBrickList = new ArrayList<>();
 
-		AssertEqualsBrick assertEqualsBrick = new AssertEqualsBrick();
-		assertionsBrickList.add(assertEqualsBrick);
-
-		WaitTillIdleBrick waitTillIdleBrick = new WaitTillIdleBrick();
-		assertionsBrickList.add(waitTillIdleBrick);
-
+		assertionsBrickList.add(new AssertEqualsBrick());
+		assertionsBrickList.add(new AssertUserListsBrick());
+		assertionsBrickList.add(new WaitTillIdleBrick());
 		assertionsBrickList.add(new TapAtBrick());
-
 		assertionsBrickList.add(new FinishStageBrick());
 
-		for (Scene scene : ProjectManager.getInstance().getCurrentProject().getSceneList()) {
-			for (Sprite sprite : scene.getSpriteList()) {
-				for (Script script : sprite.getScriptList()) {
-					for (Brick brick : script.getBrickList()) {
-						if (brick instanceof AssertEqualsBrick) {
-							assertionsBrickList.remove(assertEqualsBrick);
```

Comment:
why removing this lines

Reply:
to enable multiple assertBricks inside one project without the current workaround